src/resources/Dissolutions/index.ts

```diff
@@ -3,3 +3,5 @@ export { CommunityContributionCompanyDissolutionResource } from '@/resources/Dis
 export { CooperativeDissolutionResource } from '@/resources/Dissolutions/cooperative'
 export { LimitedCompanyDissolutionResource } from '@/resources/Dissolutions/limitedCompany'
 export { UnlimitedDissolutionResource } from '@/resources/Dissolutions/unlimitedCompany'
+
+export { soloPropCompanyDissolutionResource } from '@/resources/Dissolutions/soloPropCompany'
```

Comment:
Our style for "const" objects like this one is to PascalCase them. This is so they look like unmutable things instead of variables.